### ISSUE TITLE
Вопрос по второй лабе

### ISSUE THREAD

**Issue by fminkin** [agsh/mipt2016#9]
"Написать функцию, которая принимает JSON-объект и возвращает новый, где ключи — значения, а значения — ключи"
Верно ли, что в таком случае все ключи — строки? Иначе что делать, если ключи — не строки?

**Comment by agsh** [comment]
Да, разумеется, ключи будут строками по определению. Если ключи не строки, их можно сериализовать.
